src/main/java/pl/psnc/dei/util/RecordTransferValidator.java

```diff
@@ -0,0 +1,39 @@
+package pl.psnc.dei.util;
+
+import org.apache.jena.atlas.json.JsonObject;
+import pl.psnc.dei.model.Aggregator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RecordTransferValidator {
```

Comment:
This name of the class is not the best...actually it validates whether IIIF is available or conversion is possible...